app/views/content/landing/career-progression.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "What are the benefits of teaching?"
```

Comment:
This feels a bit long on desktop due to the way the headers split over multiple lines (which we will be reviewing in the new year!) - is there any way of shortening this?

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/cb7d8738-6951-468b-ba2a-924dd57dd739)

Reply:
Also might be worth coming up with one or more alternate headings that we can split test?

Reply:
@gemmadallmandfe I couldn't work out how to move the content to move the of on to the same line as benefits so the header is across 3 lines. So it's similar to some of the other landing page headers. I was going to ask @garethwilliamsdesign to take a look. 

But we can also look to shorten it and have a couple of different versions too.

Reply:
The advertising team have some suggestions about using a heading that ties into the campaign a bit more - they will be sending us some ad headlines as input